rust/bin-shared/src/tun_device_manager/linux.rs

```diff
@@ -61,14 +62,15 @@ impl TunDeviceManager {
     /// Creates a new managed tunnel device.
     ///
     /// Panics if called without a Tokio runtime.
-    pub fn new() -> Result<Self> {
+    pub fn new(mtu: usize) -> Result<Self> {
```

Comment:
I guess the MTU is required to be `usize` elsewhere because it's used for checking the length of slices?

Reply:
Yes

Reply:
this is a bit nitpicky but we should make `MTU` a u32 and cast it to `usize` which is infallible in any modern platform.

Reply:
Should be easier than ever after this :)